src/Map.js

```diff
@@ -147,42 +150,83 @@ function Map() {
-          <Popup closeButton={false} offset={offset}>
-            <Text textAlign={'center'}>
-              <strong>{estacion.name.toUpperCase().split('-')[1]}</strong>
-              {isNear && <>
-                <br />
-                {Math.round(nearestEstaciones[station_id].distance, 2)} metros
-                <br />
-              </>}
-            </Text>
-          </Popup>
-        </Marker>)
-      })}
-      <Marker icon={CurrentLocationMarker} position={center} />
-      <LeafletPlugins />
-
-    </MapContainer>
+    <>
+      <MapContainer
+        center={OBELISCOU}
+        maxBounds={CABA_BOUNDS}
+        maxBoundsViscosity={0.9}
+        zoom={17}
+        minZoom={13}
+        maxZoom={18}
+        style={{ height: "100vh", width: "100vw" }}
+      >
+        <IconButton
+          zIndex={1000}
+          right={2}
+          top={2}
+          variant="solid"
+          border="2px solid"
+          borderColor="gray.500"
+          colorScheme="gray"
+          position={"absolute"}
+          size="sm"
+          icon={<Icon as={BsInfo} boxSize={7} />}
+          onClick={onOpen}
+        />
+        <TileLayer url="https://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png" />
+        {Object.values(estaciones).map((estacion) => {
+          const station_id = estacion.station_id
+          const isNear = Object.keys(nearestEstaciones).includes(station_id.toString())
+          const bicis_disponibles = bicis[station_id]?.num_bikes_available
+          const color = nearestEstaciones[station_id]?.color
+          const icon = BiciIconGen(bicis_disponibles, color)
+          const offset = new Point(25, -1)
+          return (<Marker
+            icon={icon}
+            position={[estacion.lat, estacion.lon]} key={station_id}>
+            <Popup closeButton={false} offset={offset}>
+              <Text textAlign={'center'}>
+                <strong>{estacion.name.toUpperCase().split('-')[1]}</strong>
+                {isNear && <>
+                  <br />
+                  {Math.round(nearestEstaciones[station_id].distance, 2)} metros
+                  <br />
+                </>}
+              </Text>
+            </Popup>
+          </Marker>)
+        })}
+        <Marker icon={CurrentLocationMarker} position={center} />
+        <LeafletPlugins />
+      </MapContainer>
+
+      <Modal isOpen={isOpen} onClose={onClose} size="xl">
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Bicis BA</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            Este predictor de <Link color="#ec7000" href="https://baecobici.com.ar/" isExternal={true}>Ecobicis de CABA</Link> te ayuda a elegir la estación con bicicletas disponibles más cercana para evitar llegar a una estación sin bicis.
+            <br />
+            <br />
+            La idea es predecir de las estaciones que tengas cerca cuál va a tener bicicletas disponibles<Text as={"i"}> para cuando lleguemos ahí.</Text>
```

Comment:
En vez de cual es la que va a tener bicis disponible, pq capaz ninguna tiene o todas tienen, la idea es mas predecir cuan probable es q tengan bicis disponibles.
'Lleguemos ahi' suena a q es un mismo lugar, pero seria a cada estacion respectivamente.
```suggestion
            La idea es predecir para las estaciones que tengas cerca cuan probable es que tengan bicicletas disponibles <Text as={"i"}> para cuando lleguemos a ellas.</Text>
```